RELEASING.md

```diff
@@ -0,0 +1,9 @@
+# Releasing
+
+This document summarizes the process of doing a new release of this project.
+Release can only be performed by Datadog maintainers of this repository.
+
+## Schedule
```

Comment:
- Let's add that we would release for every new agent release as well (with 1/2 weeks after agent release)
- Also, let's have a section that just say that we would trigger the pipeline. Just to clear out the doubt that it is not something the community can trigger.

Reply:
I disagree, let's not tie this one to the agent since the nozzle itself is unrelated to the agent

> Release can only be performed by Datadog maintainers of this repository.

That doesn't leave doubt